code/modules/client/preference_setup/loadout/loadout_belt.dm

```diff
@@ -1,17 +1,29 @@
 /datum/gear/belt
-	display_name = "belt"
+	display_name = "belt selection"
```

Comment:
```suggestion
	display_name = "belt selection (colorable)"
```

Reply:
this is the only belt selection available so calling it colorable implies there's any alternative

Reply:
it's mostly just to let people know its colourable and not just regular belts.